js/apps/system/_admin/aardvark/APP/react/src/views/g6graphs/components/Drawer/Drawer.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ReactNode } from "react";
```

Comment:
let's use `<Drawer>` from Chakra, and remove this file

See docs: https://v1.chakra-ui.com/docs/components/overlay/drawer